discord.go

```diff
@@ -74,7 +74,6 @@ func New(args ...interface{}) (s *Session, err error) {
 	s.Identify.GuildSubscriptions = true
 	s.Identify.Properties.OS = runtime.GOOS
 	s.Identify.Properties.Browser = "DiscordGo v" + VERSION
-	s.Identify.Intents = IntentsAllWithoutPrivileged
```

Comment:
I think having this as a default is fine; it's just that having *no* option to opt-out concerned me.

Reply:
I like that too but maybe we should set it when Intent will be required. I just thought to add it when it will be required by discord.